### PR TITLE
Save current tab when selecting and open the last saved tab on refresh

### PR DESCRIPTION
**Pull request by roonyh**

### CODE REVIEW

File: modules/web/js/tab/file-tab-list.js [hunk 1]
```diff
@@ -75,12 +75,20 @@ import File from '../workspace/file';
...
                     var file = new File(fileData, {storage:self.getBrowserStorage()});
                     var tab = self.newTab(_.set({}, 'tabOptions.file', file));
                     tab.updateHeader();
                 });
+
+                if(!_.isUndefined(activeTabId)){
+                    var activeTab = _.find(this.getTabList(), function(t){
+                        return t.cid === activeTabId
+                    });
+                    this.setActiveTab(activeTab);
```
Comment (hemikak) [inline modules/web/js/tab/file-tab-list.js]: Can active tab become undefined here ?

File: modules/web/js/tab/file-tab-list.js [hunk 2]
```diff
@@ -75,12 +75,20 @@ import File from '../workspace/file';
...
+                var activeTabId = this.getBrowserStorage().get('activeTab');
                 this._workingFileSet.forEach(function(fileID){
                     var fileData = self.getBrowserStorage().get(fileID);
                     var file = new File(fileData, {storage:self.getBrowserStorage()});
                     var tab = self.newTab(_.set({}, 'tabOptions.file', file));
                     tab.updateHeader();
                 });
+
+                if(!_.isUndefined(activeTabId)){
+                    var activeTab = _.find(this.getTabList(), function(t){
```
Comment (hemikak) [inline modules/web/js/tab/file-tab-list.js]: Better to have a proper variable name instead of 't'

Comment (roonyh) [inline modules/web/js/tab/file-tab-list.js]: Agree. Done in #2041

File: modules/web/js/tab/file-tab-list.js (hunk 1 above)
Comment (roonyh) [inline modules/web/js/tab/file-tab-list.js]: Yeah. An undefined check was added in a later commit 5e4af1c36dc1f3a4b50c2e826

**Pull request merged**